tests/MaintainersTest.php

```diff
@@ -209,28 +209,53 @@ public function testListMaintainers()
         $this->assertArrayHasKey('zendeskUrl', $maintainer);
         $this->assertNull($maintainer['zendeskUrl']);
     }
-    
-    public function testUserMaintainerUsers()
+
+    public function testLeastOneMemberLimit()
```

Comment:
řekl bych spíš "AtLeast..." než "Least". Ale možná lepší "Maintainer cannot be empty" :)

Reply:
Nebo "MustHaveSomeMembers"